src/assets/google-fonts.css

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css?family=Inconsolata:400,700');
```

Comment:
you have to download this locally and add it to font, we cant reference online fonts